### PR TITLE
WAL-380

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: packages/connection-manager/__tests__/shared/connectionManagerAgentLogic.ts [hunk 1]
```diff
@@ -81,6 +81,19 @@ export default (testContext: { getAgent: () => ConfiguredAgent; setup: () => Pro
       expect(result.length).toBeGreaterThan(0)
     })
 
+    it('should get parties by filter', async () => {
```
Comment (nklomp) [inline packages/connection-manager/__tests__/shared/connectionManagerAgentLogic.ts:84]: Can we also have some testcases, which only have one property (like only name), but more importantly have a test that shouldn't return anything.

**Review by nklomp — CHANGES_REQUESTED** [review]
Small request for testcases. Rest LGTM

File: packages/connection-manager/__tests__/shared/connectionManagerAgentLogic.ts (hunk 1 above)
Comment (BtencateSphereon) [reply packages/connection-manager/__tests__/shared/connectionManagerAgentLogic.ts:84]: added additional test cases

**Review by nklomp — APPROVED** [review]

**Comment by nklomp** [comment]
LGTM

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**